### DISCUSSION TITLE
Set startup sheet

### DISCUSSION THREAD

**Discussion by mark-0lchanski** [andmarti1424/sc-im#772] [Q&A] [answered]
Hey,

my files usually have multiple sheets and whenever I add a new one, it will become the default sheet that is shown when opening the file. To change that I have to manually edit the file and change the order of the `newsheet "foo"` lines.

Is there a better way of doing that? I have tried to use `movetosheet "bar"` but that did nothing.

Regards,
Mark

**Comment by andmarti1424** [comment, marked as answer]
Hello. The default sheet to show when opening the file is the sheet where the focus was when saving the file. not the last added.
